src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -33,8 +33,7 @@ def __init__(self, ivp, gaussfilt, with_smoothing):
         if not issubclass(type(gaussfilt.dynamicmodel), ODEPrior):
             raise ValueError("Please initialise a Gaussian filter with an ODEPrior")
         self.gfilt = gaussfilt
-        self.sigma_squared_global = 0.0
-        self.sigma_squared_current = 0.0
+        self.sigma_squared_mle = 1.0
```

Comment:
aren't both sigmas kind of MLEs? (I remember you calling the time-varying one something like a quasi MLE or so). Since this current implementation is only considering the time-varying one, shall we maybe make this clear in this variable name? E.g. `sigma_square_local_mle` or sth? Thoughts?

Reply:
WIth `simga_squared_mle` I indeed mean the MLE for the _current_ sigma, but that exists for both the dynamic and time-fixed diffusion models. That is also the one that is needed for local error estimation.

In the fixed case, it corresponds to the global sigma, so I don't see a reason to have a separate variable for it. I'm not sure what to call the object that you previously stored in `sigma_squared_current` ("increment"?), but in any case: Do we actually need that anywhere? If not I think a `sigma_squared_mle` should be sufficient for both the dynamic and fixed model.

There is another open issue that's related to this topic: The sigmas need to be accounted for in the dense output in ODESolution and KalmanPosterior. In the "fixed" case that's just a number (the last MLE is the global MLE), in the "dynamic" case we need to save the full array. I'll check if we have an issue to track that and open one if not. But I think that does not have to be fixed in this PR.

Reply:
> WIth simga_squared_mle I indeed mean the MLE for the current sigma, but that exists for both the dynamic and time-fixed diffusion models. That is also the one that is needed for local error estimation.

Hmm okay, that makes sense. 

> Do we actually need that anywhere?

Not if we commit to the time-varying things for now, I guess.

Reply:
>> Do we actually need that anywhere?
> Not if we commit to the time-varying things for now, I guess.
What I mean is: Even with the time-fixed sigma, do we ever explicitly need this local increment? I'm asking because the algorithm as I described it in my paper would not use that anywhere.

Reply:
Ah I see. I guess it was like this before, because the calibration was implemented in a callback, which did not return anything. I am writing this from memory though, which could be wrong. In general I dont see that it will be necessary, unless for implementation reasons, so I agree with you

Reply:
Oh I see, I already wondered why it was implemented in a callback but I understood the reason for it only now (and I think I actually implemented that wrongly in my Julia code so thanks a lot for helping me find that bug :D). It might very well be that I'll add a callback back in again then I guess, but maybe the "increment" and "global MLE" will then be stored in some "DiffusionEstimate" object.

Reply:
sounds wonderful